akka-remote/src/main/scala/akka/remote/artery/compress/InboundCompressions.scala

```diff
@@ -404,7 +405,7 @@ private[remote] abstract class InboundCompression[T >: Null](
               resendCount = 0
               advertiseCompressionTable(association, table)
             } else
-              log.debug("Inbound compression table for originUid [{}] not changed, no need to advertise same.", originUid)
+              log.debug("{} for originUid [{}] not changed, no need to advertise same.", Logging.simpleName(tables.activeTable), originUid)
```

Comment:
is this fixed by the logger instead?

Reply:
Yes the logger was fine actually, my bad

Reply:
but the logger is still wrong, ActorRef vs Manifest:
```
val _inboundActorRefsLog = Logging(system, classOf[InboundManifestCompression])
```

Reply:
Thanks, fixed